app/models/manageiq/providers/kubernetes/container_manager/event_catcher_mixin.rb

```diff
@@ -55,17 +55,23 @@ def queue_event(event)
   end
 
   def filtered?(event)
-    extract_event_data(event).nil?
+    parsed_event = extract_event_data(event)
+    parsed_event.nil? || filtered_events.include?(parsed_event[:event_type])
```

Comment:
@Fryguy should `filtered_events.include?(parsed_event[:event_type])` be the default in the `EventCatcher::Runner` base class? (Instead of the current `false`).

If that's the case I have the feeling that we may get away with just not implementing `filtered?` at all, right?